share/completions/iw.fish

```diff
@@ -0,0 +1,275 @@
+# iw(8) completion for fish
+
+# The completions for iw are based off those for ip, which has a similar options structure.
+
+set -l iw_commands dev phy reg event features commands list
+# wdev not supported since it is barely documented
+
+function __fish_iw_device
+    __fish_print_interfaces | while read i
+        printf '%s\t%s\n' $i "Device"
+    end
+end
+
+function __fish_iw_phy
+    set -l phys /sys/class/ieee80211/*
+    echo $phys | xargs -r basename | string join \n
+end
+
+function __fish_iw_ssid
+    iw dev $argv[1] scan dump | grep SSID | awk '{ print $2 }'
```

Comment:
This could be a single `awk /SSID/ { print $2 }`, but prefer fish's `string`:

`iw dev $argv[1] scan dump | string replace --regex --filter '\s*SSID:\s*(.*)' '$1'`
Or a more direct equivalent:
`printf '%s\n' (iw dev $argv[1] scan dump | string match --regex --entire SSID | string split ' ')[2]`